pkgs/applications/misc/krename/default.nix

```diff
@@ -17,15 +17,17 @@ in mkDerivation rec {
     sha256 = "136j1dkqrhv458rjh5v3vzjhvq6dhz7k79zk6mmx8zvqacc7cq8a";
   };
 
+  buildInputs = [ taglib exiv2 podofo ];
+
+  nativeBuildInputs = [ extra-cmake-modules kdoctools wrapGAppsHook ];
+
+  propagatedBuildInputs = [ kconfig kcrash kinit kjsembed ];
+
```

Comment:
No other changes, eh?

Reply:
Indeed, "cleanup" might be a better commit message

Reply:
Not even that, there are new dependencies imported in `krename`. I would split that out and explain why in the commit message.

For the "update homepage" parts of the patchset, I'd approve... but merging stuff with the wrong commit message is a no-no for me.

:smile:

Reply:
> Not even that, there are new dependencies imported in krename. I would split that out and explain why in the commit message.

There aren't. The lines are just moved.

Reply:
Oh, that's actually true. Sorry about that! 

Still, the commit message should be more explanatory about that ("Moved some attributed for better readability" would be fine for me, I guess).

Reply:
Moving a few lines up before the meta tag doesn't change anything in terms of input (and hence output). But fair enough, I have amended the commit to mention the minor style changes.